### ISSUE TITLE
[ED][Project Plan] edit suggestion for Purpose

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-roles-responsbilities#2]
It may be a bit of an overstatement to say that all existing WAI docs do not have an audience in mind and even that most stakeholders are unaware of accessibility - those numbers seem to be changing rapidly. Suggested edit:

The purpose of the RA11y Matrix is to broaden the understanding of web professionals who work on any aspect of a project's development and maintenance cycle about the impact that every role can have on accessibility. The matrix is being developed to help stakeholders create better, more accessible solutions for the products they work on and to support their ability to quickly identify and use those WAI documents relevant to their specific role. Where gaps exist, the matrix will suggest new resources to be developed by WAI to further support accessibility practice across roles.

**Comment by bakkenb** [comment]
Yes, as I read through the Purpose the first time I was going to raise an issue that the way it is stated is a bit strong. To some degree that [Most stakeholders are unaware... and Existing WCAG documents are not designed with...] is true, or seems to be the perception we keep sharing, but I wouldn't put it in a way that is too black and white. I do like the directions Sharron is going with the rewrite. 

I don't want to spend too much time making this "perfect" as it is just in the project plan, but some of this text could in fact be "borrowed" for the overview of the resource if it is written well here and agreed upon.

**Comment by shawna-slh** [comment]
+1 to concerns with "Existing WCAG documents are not designed with specific audiences in mind...There is no guidance as to which sections they need to know and what they don't." e.g.,
* https://www.w3.org/WAI/tips/designing/
* https://www.w3.org/WAI/roles/

Certainly those are not comprehensive, and there would be great benefit to the Role/Responsibility-SC Table!

Better to word *positively* and avoid unnecessary negatives. :-)

**Comment by dboudreau** [comment]
Well, I've been known to speak my mind and not always being very politically correct... even for a Canadian. So I appreciate the adjustment in tone. And I definitely agree on trying to be more positive than negative _whenever possible_. We're completely open to such edits, please keep bringing them in.

**Comment by dboudreau** [comment]
Change made to project plan. Looks like I can't close issues myself, so I'll leave it for someone else to close.
